packages/core/src/BaseSwitch/BaseSwitch.styles.tsx

```diff
@@ -63,20 +63,38 @@ export const { staticClasses, useClasses } = createClasses("HvBaseSwitch", {
     width: "40px",
     height: "32px",
     padding: 0,
+    [`&[data-size="lg"]`]: {
+      "+.HvBaseSwitch-track": {
+        width: 48,
+        height: 22,
+      },
+      "& .HvBaseSwitch-thumb": {
+        left: -2,
+        top: 3,
+        width: 18,
+        height: 18,
+      },
+      "&.HvBaseSwitch-checked .HvBaseSwitch-thumb": {
```

Comment:
we should leverage our `$` utility - same above
```suggestion
      "&$checked $thumb": {
```